docs/elements/meta.md

```diff
@@ -12,48 +12,69 @@ maps.
 
 ## Attributes
 
-### `attribute-name`
+### `name`
 
-Description of attribute (repeat as required)
+The `name` attribute identifies the type of metadata that is being set. Possible
+values of `name` related to maps include:
 
 | Name          | Description                                          	  |
 |--------------	|--------------------------------------------------------	|
+| projection    | Set the projection for the document |
+| extent        | Define the bounds of the document in terms of corner position keywords and axis names, potentially combined with a zoom value if necessary (depending on coordinate system). |
+| cs            | Identifies the default coordinate reference system of coordinate strings found in the document. |
+| zoom          | Specify the 'native' zoom range min, max and value for displaying the document |
```

Comment:
@ahmadayubi Is it correct to describe the min/max/value of `<map-meta name="zoom" ...>` as "native??  I am thinking that the "value" value might be native, but the min/max values are the client side zoom range /bounds to display the content in, so not the "native" (server) zoom.  If you have a templated tile link, for example, the min/max on the inputs are "native" though.  amirite???